src/arrow-sdk.ts

```diff
@@ -311,28 +344,18 @@ export async function settleOptions(
 ) {
     const router = getRouterContract(version, wallet)
 
-    switch (version) {
-        case Version.V3:
-        case Version.V4:
-        case Version.COMPETITION:
-            // Check if on-chain function call would work using `callStatic`
-            try {
-                await router.callStatic.settleOptions(
-                    owner,
-                    ticker,
-                    readableExpiration
-                )
-            } catch (err) {
-                throw new Error("Settlement call would fail on chain.")
-            }
-
-            // Send function call on-chain after `callStatic` success
-            await router.settleOptions(owner, ticker, readableExpiration)
-
-            break
-        default:
-            throw UNSUPPORTED_VERSION_ERROR // Never reached because of the check in `getRouterContract`
+    try {
+        await router.callStatic.settleOptions(
+            owner,
+            ticker,
+            readableExpiration
+        )
+    } catch (err) {
+        throw new Error("Settlement call would fail on chain.")
     }
+
+    // Send function call on-chain after `callStatic` success
+    await router.settleOptions(owner, ticker, readableExpiration)
```

Comment:
Not necessary for this PR / version of the SDK, but we should add the short settlement as well. The way to implement that is to use the registry to get the short aggregator address and then to call the function here: `https://github.com/ArrowDFMs/arrowdfms-v1-core-avash/blob/main/contracts/ArrowShortAggregatorDelegate.sol#L267-L272`